backend/app/services/channels/team_selection.py

```
@@ -0,0 +1,124 @@
+            selection: Team selection to save
+        """
+        key = f"{TEAM_SELECTION_KEY_PREFIX}{user_id}"
+        try:
+            await cache_manager.set(
+                key, json.dumps(selection.to_dict()), expire=TEAM_SELECTION_TTL
+            )
+            logger.info(
+                f"[TeamSelectionManager] Saved team selection for user {user_id}: "
+                f"{selection.team_name} (id={selection.team_id})"
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)